### PR TITLE
fixed top page layout

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

Fixed styling for mobile

## Closes issue(s)

Mobile view for top page was not working

## Screenshots

If you have image share...

## Changes include

- [X] Bugfix (non-breaking change that solves an issues

## Checklist

- [X] I have tested this code on iPhone 6, X, iPad, and Galaxy S9

### CODE REVIEW

File: src/pages/Home/home.css [hunk 1]
```diff
@@ -14,17 +15,21 @@
...
 
 .ni-wa-character {
-  width: 375px;
-  margin: auto;
+  width: 100%;
 }
 
 .ni-wa {
-  font-size: 5rem;
+  margin-top: 0%;
```
Comment (daiHash) [inline src/pages/Home/home.css:22]: without the percentage is ok  => margin-top: 0;

File: src/pages/Home/home.css [hunk 2]
```diff
@@ -14,17 +15,21 @@
...
+  font-size: 3rem;
   color: #172f4a;
-  margin: 0;
 }
 
 .ni-wa-definition {
-  margin-left: 0%;
   color: #172f4a;
+  margin-left: 0%;
```
Comment (daiHash) [inline src/pages/Home/home.css:29]: same as above

**Review by daiHash — APPROVED** [review]
Nice LGTM

**Review by conalli — APPROVED** [review]
lgtm

**Pull request merged**